service/src/test/java/bio/terra/landingzone/app/TestBeanConfig.java

```diff
@@ -0,0 +1,14 @@
+package bio.terra.landingzone.app;
+
+import org.springframework.beans.factory.SmartInitializingSingleton;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class TestBeanConfig {
+  @Bean
+  public SmartInitializingSingleton postSetupInitialization(ApplicationContext applicationContext) {
```

Comment:
This is needed for spring to set up the context correctly, in order to auto-wire components. There is an equivalent class in `testharness`.